File: VB/PasswordToOpenSample/PasswordForm.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace PasswordToOpenSample
    Partial Public Class PasswordForm
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Public Property Password() As String
            Get
                Return textBox1.Text
            End Get
            Set(ByVal value As String)
                textBox1.Text = value
            End Set
        End Property
    End Class
End Namespace

File: CS/PasswordToOpenSample/PasswordForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace PasswordToOpenSample {
    public partial class PasswordForm : Form {
        public PasswordForm() {
            InitializeComponent();
        }

        public string Password {
            get { return textBox1.Text; }
            set { textBox1.Text = value; }
        }
    }
}
